src/model.js

```diff
@@ -2559,9 +2569,9 @@ Specify a different name for either index to resolve this issue.`);
 
     // Map field names
     const updatedDataValues = _.pick(instance.dataValues, changed);
-    const insertValues = Utils.mapValueFieldNames(instance.dataValues, Object.keys(instance.rawAttributes), this);
-    const updateValues = Utils.mapValueFieldNames(updatedDataValues, options.fields, this);
-    const now = Utils.now(this.sequelize.dialect);
+    const insertValues = mapValueFieldNames(instance.dataValues, Object.keys(instance.rawAttributes), this);
+    const updateValues = mapValueFieldNames(updatedDataValues, options.fields, this);
+    const now = new Date();
```

Comment:
`now` is used in two places, I think we can just as well put `new Date()` there

Reply:
You mean instead of using this variable? We could get two sightly different dates for what is supposed to be the same instant if we do that

Reply:
That's a good point